phpcs-sample.xml

```diff
@@ -94,4 +94,18 @@
             </property>
         </properties>
     </rule>
+
+    <rule ref="LipematWordpress">
+        <!-- Exclude overridden Sniffs -->
+        <exclude name="WordPressVIPMinimum.JS.DangerouslySetInnerHTML"></exclude>
```

Comment:
Won't need to exclude this here after the sniffs are complete because we simply won't include `WordPressVIPMinimum.JS`

Reply:
`Wordpress-VIP-Go` is including all `WordpressVIPMinimum` sniffers, so we have to exclude it, otherwise will be included by that package: https://github.com/Automattic/VIP-Coding-Standards/blob/develop/WordPress-VIP-Go/ruleset.xml#L7

I think we can exclude it in our new custom sniffer using the `ruleset.xml`, that way we don't need to add this `exclude` rule in all the projects that going to be used in the `phpcs.xml` file.

Reply:
We don't want to exclude any sniffs with the `Lipe` ruleset because this library is not project specific and will be conflicting for other non VIP-Go project.
The rule may be excluded per project like so.

```xml
<rule ref="WordPressVIPMinimum.JS">
        <exclude name="WordPressVIPMinimum.JS.DangerouslySetInnerHTML.Found"></exclude>
        <type>error</type>
</rule>
```

Although, once this feature is complete, there will be no reason to include `WordPressVIPMinimum.JS` so we will simply exclude via.

```xml
<rule ref="WordPress-VIP-Go">
        <exclude name="WordPressVIPMinimum.JS" />
</rule>
```